### PR TITLE
update README to remove paloma-testnet-7 and clean up again

### PR DESCRIPTION
**Pull request by taariq**
# Related Github tickets

- Cleaning up the README again.

# Background

Readme had paloma testnet-7 to testnet-8 upgrade instructions still hanging.

# Testing completed
Please check

# Breaking changes
None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,10 +52,9 @@ We have active, helpful communities on Twitter and Telegram.
 See [Release procedure](CONTRIBUTING.md#release-procedure) for more information about the release model.
 
 ## Active Networks
+### Testnet: Paloma-Testnet-9
```
Comment (measure-fi) [inline README.md:55]:
```
`paloma-testnet-9`
```

File: README.md [hunk 2]
```diff
@@ -82,7 +81,7 @@ wget -O - https://github.com/palomachain/paloma/releases/download/v0.8.0/paloma_
   tar -C /usr/local/bin -xvzf - palomad
 ```
 
-2. Setup your pigeon by following [the instructions](https://github.com/palomachain/pigeon#install) and then come back here.
+2. Setup your pigeon relayer by following [the instructions](https://github.com/palomachain/pigeon#install) and then come back here.
```
Comment (measure-fi) [inline README.md:84]: Extremely minor, but if you link a pertinent part of the sentence `[Set up your pigeon relayer](linky) by following the linked instructions` it's much easier for screen readers to tab through and pick it up.

File: README.md [hunk 3]
```diff
@@ -104,28 +103,7 @@ wget -O ~/.paloma/config/addrbook.json https://raw.githubusercontent.com/palomac
...
-3. Copy the latest genesis and addrbook
-```shell
-wget -O ~/.paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/testnet/master/paloma-testnet-8/genesis.json
-wget -O ~/.paloma/config/addrbook.json https://raw.githubusercontent.com/palomachain/testnet/master/paloma-testnet-8/addrbook.json
-```
-
-4. Ensure that the [latest pigeon](https://github.com/palomachain/pigeon#install) is up & running and that you have at least 0.1 ETH on eth mainnet target chain. Make sure to change the `chain-id: paloma-testnet-7` to `chain-id: paloma-testnet-8` in your pigeon config yaml file ([example](https://github.com/palomachain/pigeon/pull/51/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R85)).
-
-5. Start the palomad
+7. If you are planning to be a **VALIDATOR with stake**, ensure that your pigeon relayer is up & running and that you have at least 0.1 ETH on eth mainnet target chain address that is in your pigeon configuration file.
```
Comment (measure-fi) [inline README.md:106]: Why are all those lines deleted?

**Review by measure-fi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (taariq) [reply README.md:55]: Updated.

File: README.md (hunk 2 above)
Comment (taariq) [reply README.md:84]: Agreed. Updated.

File: README.md (hunk 3 above)
Comment (taariq) [reply README.md:106]: Because those lines are already covered in the pigeon README instructions here: https://github.com/palomachain/pigeon#config-setup

**Review by Vizualni — APPROVED** [review]
LGTM

**Pull request merged**